pkgs/top-level/python-packages.nix

```diff
@@ -29464,6 +29464,25 @@ EOF
     };
   };
 
+  pywbem = buildPythonPackage rec {
+    name  = "pywbem-${version}";
+    version = "0.10.0";
+    src = pkgs.fetchFromGitHub {
+      owner = "pywbem";
+      repo = "pywbem";
+      rev = "v${version}";
+      sha256 = "0jcwklip03xcni0dvsk9va8ilqz21g4fxwqd5kzvv91slaadfcym";
+    };
+    propagatedBuildInputs = with self; [ m2crypto ply pyyaml six ];
+    doCheck = false;
```

Comment:
For some reason running ```make test``` works but setting ```checkTarget = "test";``` doesn't. In any case, test are now in place.